reporting/reporting_sierra_transformer/src/test_transform.py

```diff
@@ -0,0 +1,121 @@
+import json
+import datetime
+from copy import deepcopy
+from transform import transform, unpack
+
+
+def build_test_data(full=False):
+    bib_record = {
+        "string_field": "some string",
+        "boolean_field": True,
+        "single_dict_single_subfield": {"a": "b"},
+        "single_dict_multiple_subfields": {"a": "b", "x": "y"},
+        "multiple_dicts_single_subfield": [{"a": "b"}, {"a": "c"}],
+        "multiple_dicts_multiple_subfields": [
+            {"a": "b", "x": "y"},
+            {"a": "c", "x": "z"},
+        ],
+        "orders_date": ["1970-01-01T00:00:00"],
+        "publishYear": 1970,
+        "empty_string": "",
+    }
+
+    if full:
```

Comment:
Rather than an ambiguously named `full` parameter, how about splitting this into two methods?

```python
def build_bib_record()

def build_sierra_transformable()
```